prospector/filter_rank/utils/model_loader.py

```diff
@@ -0,0 +1,21 @@
+from filter_rank import MODELS_FOLDER
```

Comment:
This module looks good!

We need the one to make models now :-)

Reply:
I also added some meat to this model, so we can consider it more or less done for the moment.

Switching to the model-maker part :)